src/V3ParseImp.cpp

```diff
@@ -528,10 +528,13 @@ void V3ParseImp::tokenPipelineSym() {
                     token = yaID__ETC;
                 }
             }
+        } else if (token == yaID__CC && *(yylval.strp) == "std") {  // IEEE-standard package
+            v3Global.setUsesStdPackage();
```

Comment:
Will this break if the user does "foo::std" where std is a class inside Foo package, this I believe is legal.  Add a test?

Reply:
It will "break" as in it will make us parse `std.sv`, but it'll still verilate. I think it's a rare enough case that it should be fine?

Reply:
It will skip the other if statement branches below.  Maybe there's nothing harmful in that, but still seems worthy of a test to make sure.

Reply:
Ah, I see. I'll add a test.

But the `else` should be executed in that `foo::std` case. Would something like:

```cpp
} else {  // Not found
    yylval.scp = nullptr;
    if (token == yaID__CC) {
        if (m_lexPrevToken != yaID__CC && *(yylval.strp) == "std") {
            v3Global.setUsesStdPackage();
        } else if (!v3Global.opt.bboxUnsup()) {
```
make sense? Seems to be working.

Reply:
That seems reasonable.  All accesses should be a std:: or an import, both would be of a ID_CC.

Reply:
It turned out to be more complicated. The tests dindn't cover `std::` usage properly.